packages/esm-service-queues-app/src/patient-search/visit-form/queue.resource.ts

```diff
@@ -27,20 +26,12 @@ export async function addQueueEntry(
     },
     signal: abortController.signal,
     body: {
-      visit: { uuid: visitUuid },
+      visit: visitUuid,
       queueEntry: {
-        status: {
-          uuid: status,
-        },
-        priority: {
-          uuid: priority,
-        },
-        queue: {
-          uuid: queueServiceUuid,
-        },
-        patient: {
-          uuid: patientUuid,
-        },
+        status: status,
+        priority: priority,
+        queue: queueServiceUuid,
+        patient: patientUuid,
         startedAt: new Date(),
```

Comment:
I was getting errors without making the above changes.  I'm not sure why we were submitting like we were before, but with these changes I'm able to get things posted correctly.